app/views/exchange_rates/_document_detail.html.erb

```diff
@@ -12,21 +12,18 @@
         <%= Date::MONTHNAMES[period.month] %> <%= period.year %> <%= type %> exchange rates
       </h3>
 
-      <p>
-        <strong>
-          <% if period.has_exchange_rates %>
-            <%= link_to "View online", exchange_rate_collection_path("#{period.year}-#{period.month}", type: type), class: 'govuk-link' %>
-
-          <% end %>
-        </strong>
+      <p class="document-download">
+        <% if period.has_exchange_rates %>
+          <%= link_to "<strong>View online</strong>".html_safe, exchange_rate_collection_path("#{period.year}-#{period.month}", type: type), title: "View #{period.month} #{period.year} monthly exchange rates online", class: 'govuk-link' %>
+        <% end %>
 
         <% if period.files.any? %>
-          <%= " | " if period.has_exchange_rates %>
-          <% period.files.each_with_index do |file, index| %>
-            <strong><a href="<%= file.file_path %>"><%= file.format.upcase %></strong> <%= file.file_size_in_kb %> KB</a>
-              <%= ' | ' unless index == period.files.size - 1 %>
-            <% end %>
+          <% period.files.each do |file| %>
+            <a title="Download <%= period.month%> <%= period.year %> monthly exchange rates in <%= file.format.upcase %> format" href="<%= file.file_path %>">
```

Comment:
Its probably worth using the `link_to` helper - it means when rails adds additional attributes to `<a>` tags for security purposes, then links such as this will be benefit - you can still use code blocks to separate the 'linked' content from the link_to helper, eg

```erb
<%= link_to file.file_path title: "Download <% ... %>" do %>
  <strong><%= file.format.upcase %></strong>
  <%= file.file_size_in_kb %>
<% end %>
```